prep-css-flexbox/display-flex.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+    }
+    card-deck {
+      padding: 10px;
+    }
```

Comment:
`card-deck` by itself won't actually select any elements. `card-dec` would be looking for an element named `<card-deck>` which doesn't exist. You want to combine these 2 rulesets into 1 rule set, just with the `id` selector:

```CSS
#card-deck {
  display: flex;
  padding: 10px;
}
```